bigquery_exporter/base.py

```diff
@@ -46,10 +52,35 @@ def __init__(self):
                 raise ValueError(f'Custom field {field} is not defined')
```

Comment:
Need to remove this for-loop since self.custom_fields no longer exists

Reply:
no longer relevant with commit revert

Reply:
Sorry, erroneous commit there. Reverted and will implement the custom_fields decorator in a follow-up PR.